src/libmscore/draw/drawjson.cpp

```diff
@@ -197,7 +197,7 @@ static void fromObj(const QJsonObject& obj, QPainterPath& path)
 
     QJsonArray elsArr = obj["elements"].toArray();
     std::vector<QPainterPath::Element> curveEls;
-    for (const QJsonValue& elVal : elsArr) {
+    for (const QJsonValue elVal : elsArr) {
```

Comment:
Can you please explain why you removed references?

Reply:
Clang says: "Loop variable 'elVal' is always a copy because the range of type 'QJsonArray' does not return a reference"